tests/framework/extensions/rke1/componentchecks/etcdversion.go

```diff
@@ -23,21 +21,19 @@ func CheckETCDVersion(client *rancher.Client, nodes []*nodes.Node, clusterID str
 	var etcdResult []string
 
 	for _, rancherNode := range nodesList.Data {
-		externalIP := rancherNode.Annotations["rke.cattle.io/external-ip"]
-		etcdRole := rancherNode.Labels["node-role.kubernetes.io/etcd"]
-
-		if etcdRole == "true" {
+		etcdRole := rancherNode.Labels["node-role.kubernetes.io/etcd"] == "true"
+		if etcdRole {
 			for _, node := range nodes {
-				if strings.Contains(node.PublicIPAddress, externalIP) {
-					command := "docker exec etcd etcdctl version"
-					output, err := node.ExecuteCommand(command)
-					if err != nil {
-						return []string{}, err
-					}
-
-					etcdResult = append(etcdResult, output)
-					logrus.Infof(output)
+				command := "docker exec etcd etcdctl version"
```

Comment:
will this work on multi-node clusters?

Reply:
I will check. We only need to run this on one etcd node, hence the break. But I will verify that and follow back.

Reply:
Resolved, running new job.